CHANGELOG.rst

```diff
@@ -47,6 +48,9 @@ Changed
 - Updated Mattermost connector documentation to be more clear.
 - Updated format strings to f-strings where appropriate.
 - Updated tensorflow requirement to ``1.15.0``
+
```

Comment:
what about this line?
```suggestion
```

Reply:
not sure if you had the latest so I just pushed up everything I had so this is probably old now.